elastic/kibana
# feat(slo): add public access

Resolves https://github.com/elastic/kibana/issues/165525

## Summary

This PR makes the SLO endpoints public for serverless.


## How to test

Create the following roles:
**sre** with the `manage_transform` cluster privilege and the `all` privilege on the `.slo-*`  index pattern. you'll need to give `read` and `view_metadata` to all index you want to create SLO upon. You can use `*` index pattern for this.
Then give the `All` privilege for the SLO feature under Observability.

**sre_read** with the `read` privilege on the `.slo-*`  index pattern. 
Then give the `Read` privilege for the SLO feature under Observability.

Create one user for each previous role created.

When logging with the sre role, you should be able to create SLOs, delete, update, etc..
When logging with the sre_read role, you should be able to view the existing SLO
